content/integrate/redis-data-integration/_index.md

```diff
@@ -26,40 +25,89 @@ RDI's purpose is to help Redis customers sync Redis Enterprise with live data fr
+{{</note>}}
 
-* [**Ingest scenario**]({{< relref "/integrate/redis-data-integration/ingest" >}}): RDI mirrors the application's
-  primary database to Redis using a *change data capture* (CDC) tool. RDI transforms the database model and
-  types to a Redis model and types. This scenario is useful when the application database is not performant
-  and scalable enough to serve the read queries. RDI helps you offload all read queries to Redis.
-
-  {{<note>}}
-  Ingest is supported with Redis database or [CRDB](https://redis.com/redis-enterprise/technology/active-active-geo-distribution/) (Active Active Replication) targets.
-  {{</note>}}
-
-  {{< image filename="/images/rdi/ingest.png" >}}
-  
-* [**Write-behind scenario** (Preview)]({{< relref "/integrate/redis-data-integration/write-behind" >}}): RDI applies data
-  changes in Redis to one or more downstream data stores. RDI can map and transform Redis types and models to downstream types and models. This scenario is useful when the application needs fast writes and reads for some of the queries, but has to provide data to other downstream services that need them in different models for other uses.
-
-  {{<note>}}
-  Write-behind is *not* supported with [CRDB](https://redis.com/redis-enterprise/technology/active-active-geo-distribution/) (Active Active Replication)
-  {{</note>}}
-
-  {{< image filename="/images/rdi/write-behind.png" >}}  
-
-See the 
-[Ingest architecture guide]({{< relref "/integrate/redis-data-integration/ingest/architecture" >}})
-and the
-[Write-behind architecture guide]({{< relref "/integrate/redis-data-integration/write-behind/architecture" >}})
-for more information.
-
-## Supported sources (ingest)
+## Features
 
-RDI supports the following database sources using [Debezium Server](https://debezium.io/documentation/reference/stable/operations/debezium-server.html) connectors:
+RDI provides enterprise-grade streaming data pipelines with the following features:
+
+- **Near realtime pipeline** - The CDC system captures changes in very short time intervals,
+  then ships and processes them in *micro-batches* to provide near real time updates to Redis.
+- **At least once guarantee** - RDI will deliver any change to the selected data set at least
+  once to the target Redis database.
+- **Data integrity** - RDI keeps the data change order per source table or unique key.
+- **High availability** - All stateless components have hot failover or quick automatic recovery.
+  RDI state is always highly available using Redis Enterprise replication.
+- **Easy to install and operate** - Use a self documenting command line interface (CLI)
+  for all installation and day-two operations.
+- **No coding needed** - Create and test your pipelines using Redis Insight.
+- **Data-in-transit encryption** - RDI never persists data to disk. All data in-flight is
```

Comment:
This may be a UK vs US english thing. Feel free to ignore.

```suggestion
- **Data-in-transit encryption** - RDI never persists data to disk. All in-flight data is
```